src/components/ui/ContextMenuDropdown.tsx

```diff
@@ -0,0 +1,532 @@
+            }}
+          >
+            {item.subitems.map((subitem, subitemIndex) => (
+              <button
+                key={`subitem-${subitemIndex}`}
+                role="menuitem"
+                aria-label={subitem.text}
+                id={`${dropdownId}-subitem-${index}-${subitemIndex}`}
+                className={mergeClasses(
+                  'flex items-center px-2 py-1.5 hover:bg-white/5 text-white text-xs cursor-pointer w-full',
+                  focusedSubIndex === subitemIndex && focusedSubIndex !== -1 ? 'bg-white/10' : ''
+                )}
+                onMouseDown={(e) => e.preventDefault()}
+                onClick={(e) => handleSubItemClick(e, subitem.action, subitem.data)}
+                onKeyDown={(e) => handleSubItemKeyDown(e, subitem.action, subitem.data)}
+                tabIndex={-1}
+              >
+                <p>{subitem.text}</p>
+              </button>
+            ))}
+          </div>
+        )}
+      </div>
+    ) : (
+      <button
+        key={index}
+        role="menuitem"
+        aria-label={item.text}
+        id={`${dropdownId}-item-${index}`}
+        className={mergeClasses(
+          'flex items-center px-2 py-1.5 hover:bg-white/5 text-white text-xs cursor-pointer w-full',
+          focusedIndex === index && focusedSubIndex === -1 ? 'bg-white/10' : ''
+        )}
+        onClick={(e) => handleItemClick(e, item.action)}
+        onKeyDown={(e) => handleItemKeyDown(e, item.action)}
+        onMouseDown={(e) => e.preventDefault()}
+        tabIndex={-1}
+      >
+        <p className="text-left">{item.text}</p>
+      </button>
+    )
+  )
+
+  return (
+    <div cy-id="wrapper" className={mergeClasses('relative h-9 w-9', className)}>
+      {!processing ? (
+        <button
+          ref={buttonRef}
+          type="button"
+          disabled={disabled}
+          className="relative h-full w-full flex items-center justify-center shadow-xs pl-3 pr-3 text-left cursor-pointer text-gray-100 hover:text-gray-200 rounded-full hover:bg-white/5"
+          aria-label="More options"
+          aria-haspopup="true"
+          aria-expanded={showMenu}
+          aria-controls={dropdownId}
+          aria-activedescendant={
+            focusedSubIndex !== -1 && focusedSubIndex >= 0 && openSubMenuIndex !== null
+              ? `${dropdownId}-subitem-${openSubMenuIndex}-${focusedSubIndex}`
+              : focusedIndex >= 0
+              ? `${dropdownId}-item-${focusedIndex}`
+              : undefined
+          }
+          onClick={handleButtonClick}
+          onKeyDown={handleKeyDown}
+        >
+          <span className={`material-symbols text-2xl ${iconClass}`}>&#xe5d4;</span>
+        </button>
+      ) : (
+        <div className="h-full w-full flex items-center justify-center">
+          <LoadingSpinner />
+        </div>
+      )}
+
+      <CSSTransition
+        in={showMenu}
+        timeout={{
+          enter: 200,
```

Comment:
I'm thinking we should have the enter transition 0.
We lose the transition but it feels much snappier and I hardly notice that it isn't transitioning. When I put a few of these side by side with different values the one with 0 felt much better.

Reply:
You're right these aren't working in the nuxt app. My eyes were tricking me.
I did some testing with the transitions and I agree we should remove them from dropdowns.